ta/Makefile

```diff
@@ -4,6 +4,8 @@ CFG_TEE_TA_LOG_LEVEL ?= 1
 
 CPPFLAGS += -Iinclude
 
+CFLAGS += -DCFG_CACHE_API
+
```

Comment:
On i.MX8M, we have disabled the CFG_CACHE_API in OP-TEE because we have mapped the secure heap as **uncached**. Can you make this configurable using an environment variable?